Model/Import/Category.php

```diff
@@ -797,11 +616,10 @@ protected function indexDeleteEvents()
      * @return $this
      * @throws \Exception
      */
-    protected function reindexUpdatedCategories($categoryId)
+    private function reindexUpdatedCategories($categoryId)
```

Comment:
Is there a good reason why this should be `private`? We override this method in a project to disable indexing each category separately, because this is slow as hell. I think this is a good use case to keep this `protected` :)

Reply:
Reverting it to `protected`.